### PR TITLE
chore: enable plugins & add address alias mdx component

### PR DESCRIPTION
**Pull request by fionnachan**
#### Changes
- update the sdk pin
- enable mermaid and solidity syntax highlighting
- add address alias mdx component

### Before
https://developer.arbitrum.io/proving/challenge-manager
image

### After
https://nitro-docs-git-enable-plugins-offchain-labs.vercel.app/proving/challenge-manager
image

### Before
https://developer.arbitrum.io/arbos/l1-to-l2-messaging#address-aliasing
image

### After
https://nitro-docs-git-enable-plugins-offchain-labs.vercel.app/arbos/l1-to-l2-messaging#address-aliasing
image

## Address Alias MDX Component
https://nitro-docs-git-enable-plugins-offchain-labs.vercel.app/arbos/l1-to-l2-messaging#address-aliasing
image

Fixes https://github.com/OffchainLabs/arbitrum-docs/issues/39

### CODE REVIEW

File: arbitrum-docs/arbos/l1-to-l2-messaging.mdx [hunk 1]
```diff
@@ -1,3 +1,32 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { useEffect, useState } from "react";
+
+export const AddressAliasHelper = () => {
+  const CONSTANT = BigNumber.from("0x1111000000000000000000000000000000001111");
```
Comment (DZGoldman) [inline arbitrum-docs/arbos/l1-to-l2-messaging.mdx:5]: Let's instead use the `Address.applyAlias` method from the arbitrum-sdk (that will among other things handle the appropriate address validation; e.g., as is, the page errors on a non-hex alphanumerical input)

File: arbitrum-docs/arbos/l1-to-l2-messaging.mdx [hunk 2]
```diff
@@ -1,3 +1,32 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { useEffect, useState } from "react";
+
+export const AddressAliasHelper = () => {
```
Comment (DZGoldman) [inline arbitrum-docs/arbos/l1-to-l2-messaging.mdx:4]: Can we keep all the react components in src/components and import them into the mdx files? For cleanliness and b/c we may went to reuse them :)

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -13,7 +13,10 @@ const config = {
   onBrokenLinks: "ignore",
   onBrokenMarkdownLinks: "warn",
   favicon: "img/favicon.ico",
-
+  markdown: {
+    mermaid: true,
```
Comment (DZGoldman) [inline website/docusaurus.config.js:17]: <3

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -132,54 +134,55 @@ const config = {
...
       },
       announcementBar: {
         id: "support_us",
         content: `Note: Arbitrum is in mainnet beta, which includes trusted admin controls; use at your own risk! See <a rel="noopener noreferrer" href="/mainnet-beta">here</a> for details.`,
         backgroundColor: "rgb(121 241 3)",
         textColor: "#091E42",
-        isCloseable: true
-      }
-    })
+        isCloseable: true,
```
Comment (DZGoldman) [inline website/docusaurus.config.js:183]: what's applying all these formatting/linting changes? If we're going to do a repo wide lint, can we include whatever it is into the package json for consistency? (eslint etc.)

**Review by DZGoldman — CHANGES_REQUESTED** [review]
pumped

File: arbitrum-docs/arbos/l1-to-l2-messaging.mdx (hunk 1 above)
Comment (fionnachan) [reply arbitrum-docs/arbos/l1-to-l2-messaging.mdx:5]: oh haha didn't know there's this method

File: website/docusaurus.config.js (hunk 4 above)
Comment (fionnachan) [reply website/docusaurus.config.js:183]: it's my local prettier
can have a new PR for prettier

**Review by DZGoldman — APPROVED** [review]
lgtm!

**Pull request merged**